indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -191,6 +192,19 @@ def _process(self):
         return '', 201
 
 
+class RHDeleteEditable(RHContributionEditableBase):
+    """Delete a contribution's editable."""
+
+    def _check_access(self):
+        RHContributionEditableBase._check_access(self)
+        if not self.contrib.can_manage(session.user):
```

Comment:
I don't think this should be possible for someone who can just manage the contribution. Should require full event management privileges IMHO